armadillo/src/main/resources/application-big-files.yml

```diff
@@ -0,0 +1,8 @@
+spring:
+  mvc:
+    async:
+      request-timeout: 36000000
+  servlet:
+    multipart:
+      max-file-size: 13000MB
+      max-request-size: 13000MB
```

Comment:
This came from @SietsmaRJ local config .. need to check for the docs whether we need this in here.

Reply:
I think this should be moved to default application yaml?

Reply:
This is now in `application.yml` and `application.template.yml` for override which looks good to me.